addons/point_of_sale/static/src/backend/pos_kanban_view/pos_kanban_view.js

```diff
@@ -64,20 +64,31 @@ export class PosKanbanRenderer extends KanbanRenderer {
         this.posState.show_predefined_scenarios = this.props.list.count === 0;
     }
 
+    showAccessDeniedDialog(body) {
+        this.dialog.add(AlertDialog, {
+            title: _t("Access Denied"),
+            body: body,
+        });
+    }
+
     async callWithViewUpdate(func) {
         try {
             const isPosManager = await user.hasGroup("point_of_sale.group_pos_manager");
             if (!isPosManager) {
-                this.dialog.add(AlertDialog, {
-                    title: _t("Access Denied"),
-                    body: _t(
-                        "It seems like you don't have enough rights to create point of sale configurations."
-                    ),
-                });
+                this.showAccessDeniedDialog(
+                    _t(
+                        "It seems like you don't have enough rights to create " +
+                            "point of sale configurations."
```

Comment:
```suggestion
                        "It seems like you don't have enough rights to create point of sale configurations."
```

don't cut the original string into two.